python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -544,6 +568,11 @@ def recover_if_needed(self, node_id, now):
         logger.warning("StandardAutoscaler: "
                        "{}: No recent heartbeat, "
                        "restarting Ray to recover...".format(node_id))
+        self.event_summarizer.add(
+            "Restarting raylet on {} nodes of type " +
+            self._get_node_type(node_id) + " (heartbeat timeout).",
```

Comment:
The user might not be familiar with "heartbeat". Can we use head node cannot reach worker?